src/model/resources.py

```diff
@@ -124,10 +121,7 @@ def model_simulate():
         except ModelNotFound as error:
             return error.message, 404
 
-        # Make a copy of the shared model instance for this request. It is not sufficient to use the cobra model context
-        # manager here, as long as we're using async gunicorn workers and app state can be shared between requests.
-        # This is an expensive operation, it can take a few seconds for large models.
-        model = model_wrapper.model.copy()
+        model = model_wrapper.model
```

Comment:
Has nothing to do with this PR but I'm wondering about the following `elif` condition. Should we actually allow a client to send a full model? Now that we allow upload I'm favor of removing that and relying on models from storage. Especially since we can then rely on models having undergone a validation pipeline in future:

upload -> schema validation -> cobrapy deserialization -> memote testing -> identifier mapping -> storage

Reply:
Makes sense to me, sounds like this will help avoid potential issues with identifiers and model quality in general.